### PR TITLE
Harvest normalization failure: fixing the schemas

### LINKED ISSUE #5473: Source Harvest: failed in normalization step
## Enviroment
Is this your first time deploying Airbyte: YES
Airbyte version: 0.29.4-alpha
OS Version / Instance: Ubuntu 18.04
Deployment: Docker
Source Connector and version: Harvest 	0.1.3
Destination Connector and version: BigQuery 0.3.12
Severity: High
Step where error happened: Sync job

## Current Behavior
```
2021-08-12 03:14:56 INFO () LineGobbler(voidCall):85 - Database Error in model invoice_payments (models/generated/airbyte_tables/src_harvest/invoice_payments.sql)
2021-08-12 03:14:56 INFO () LineGobbler(voidCall):85 -   Bad int64 value: 384.0
2021-08-12 03:14:56 INFO () LineGobbler(voidCall):85 -   compiled SQL at ../build/run/airbyte_utils/models/generated/airbyte_tables/src_harvest/invoice_payments.sql
2021-08-12 03:14:56 INFO () LineGobbler(voidCall):85 -
2021-08-12 03:14:56 INFO () LineGobbler(voidCall):85 - Database Error in model invoices (models/generated/airbyte_tables/src_harvest/invoices.sql)
2021-08-12 03:14:56 INFO () LineGobbler(voidCall):85 -   Bad int64 value: 20.0
2021-08-12 03:14:56 INFO () LineGobbler(voidCall):85 -   compiled SQL at ../build/run/airbyte_utils/models/generated/airbyte_tables/src_harvest/invoices.sql
2021-08-12 03:14:56 INFO () LineGobbler(voidCall):85 -
2021-08-12 03:14:56 INFO () LineGobbler(voidCall):85 - Done. PASS=16 WARN=0 ERROR=2 SKIP=2 TOTAL=20

```

## Expected Behavior
*Tell us what should happen.*

## Logs

[logs-5-2.txt](https://github.com/airbytehq/airbyte/files/7001941/logs-5-2.txt)

<details>
<summary>LOG</summary> 

```

replace this with
your long log
output here

```

</details>

## Steps to Reproduce
1.
2.
3.

## Are you willing to submit a PR?

Remove this with your answer.

### PR DESCRIPTION
**Pull request by vovavovavovavova**
## What
Closes #5473 

## How
By fixing the schema types. In most cases these are float numbers, that should be instead of integer, but there are also other changes (like object or array detalization). Changes were made by manual comparing the schema_generator_output and existing schemas, to avoid losing nulls everywhere.

## Recommended reading order
1. `x.java`
2. `y.python`

## Pre-merge Checklist
Expand the relevant checklist and delete the others. 

<details><summary> <strong> New Connector </strong></summary>
<p>
   
#### Community member or Airbyter
   
- [ ] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] Connector's `README.md`
    - [ ] `docs/SUMMARY.md`
    - [ ] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [ ] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] Connector added to connector index like described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
   
#### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items. 
   
- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci). 
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing. 
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
   
</p>
</details>

<details><summary> <strong> Updating a connector </strong></summary>
<p>
   
#### Community member or Airbyter
   
- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] Connector's `README.md`
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] Connector version bumped like described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
   
#### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items. 
   
- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci). 
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing. 
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</p>
</details>

<details><summary> <strong> Connector Generator </strong> </summary>
<p>
   
- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- [ ] The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- [ ] Documentation which references the generator is updated as needed.
</p>
</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-harvest/source_harvest/streams.py [hunk 1]
```diff
@@ -455,3 +464,4 @@ class ProjectBudget(ReportsBase):
     """
 
     report_path = "project_budget"
+    primary_key = None
```
Comment (keu) [inline airbyte-integrations/connectors/source-harvest/source_harvest/streams.py:467]: don't you think this is common for all ReportsBase streams?
lets just move it there

**Review by keu — CHANGES_REQUESTED** [review]
see my comments

**Review by keu — APPROVED** [review]

**Review by tuliren — APPROVED** [review]

**Pull request merged**